docs/zh/navigation.yml

```
@@ -487,6 +487,8 @@ nav:
                       - Spiderpool 安装: network/modules/spiderpool/install/install.md
                       - RDMA 环境准备及安装: network/modules/spiderpool/install/rdmapara.md
                       - 安装 Nvidia OFED 驱动: network/modules/spiderpool/install/ofed_driver.md
+                      - AI 集群基于 Macvlan 技术给容器提供 RDMA 通信能力:  network/modules/spiderpool/install/ai_cluster_with_macvlan.md
```

```suggestion
                      - 为 AI 集群提供 RDMA 能力:  network/modules/spiderpool/install/ai_cluster_with_macvlan.md
```
左侧的 nav 空间狭窄，能缩短一些吗。正文的 title 可以很长

---

done